libraries/motor2040/motor2040.hpp

```diff
@@ -49,6 +49,9 @@ namespace motor {
 
     const uint USER_SW = 23;
 
+    const uint I2C_SDA = 20;
```

Comment:
Can these lines be put before USER_SW, so they are in number order?
Also `const uint I2C_INT = 19;` should be included too

Reply:
`I2C_INT` added and reordered in  [3e81b24](https://github.com/pimoroni/pimoroni-pico/pull/890/commits/3e81b245a1a16e3d791bf92210b822b52a7791ef)
`i2c_*` pin defs added to `servo2040.hpp` in [9e6a072](https://github.com/pimoroni/pimoroni-pico/pull/890/commits/9e6a0725c03daaaa8777a4e798c8c13fabf61000)